src/main/java/com/simplythewest/utd/priorityList/ui/AddMenu.java

```diff
@@ -0,0 +1,122 @@
+package com.simplythewest.utd.priorityList.ui;
+
+import com.simplythewest.utd.priorityList.models.ListManager;
+import com.simplythewest.utd.priorityList.models.ToDoDTO;
+import com.simplythewest.utd.priorityList.models.ToDoItem;
+import com.simplythewest.utd.priorityList.models.ToDoItemRepository;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.radiobutton.RadioButtonGroup;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.router.Route;
+import org.springframework.data.domain.Sort;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Route("AddMenu")
+public class AddMenu extends VerticalLayout {
+
+    private ToDoDTO toDoItem;
+    private ToDoItemRepository myToDoRepo;
+    private Grid<ToDoItem> myList;
+
+    public AddMenu(ToDoItemRepository otherToDoRepo)
+    {
+        myToDoRepo = otherToDoRepo;
+        toDoItem = new ToDoDTO();
+        myList = new Grid<>();
+
+        List<ToDoItem> theList = otherToDoRepo.findAll(Sort.by("priority").ascending());
+
+        if (!theList.isEmpty())
```

Comment:
why do we need to do this? you can have a list of empty items

Reply:
I'm getting nullpointer exceptions